lib/bosh/workspace/project_deployment.rb

```diff
@@ -44,7 +44,13 @@ def manifest
     def stub
       return @stub unless @stub.nil?
       stub_file = File.expand_path(File.join(file_dirname, "../stubs", file_basename))
-      @stub = File.exist?(stub_file) ? Psych.load(File.read(stub_file)) : {}
+      @stub = File.exist?(stub_file) ?
+        Psych.load(File.executable?(stub_file) ? execute_stub(stub_file) : File.read(stub_file))
+      : {}
```

Comment:
Would be nice to extract some of the logic here into separate methods, and don't use single line if statements when doing multiline